### PR TITLE
Add form section

### PR DESCRIPTION
**Pull request by ManarOmari**
Add form section to portfolio to complete the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -259,19 +259,41 @@ <h3>Language</h3>
...
           </div>
-         
+
         </li>
       </ul>
-    
+      <section class="form-section" >
+        <h1> Contact me</h1>
+        <p> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        <form method="post" action="https://formspree.io/html/">
```
Comment (descholar-ceo) [inline index.html]: - [x] The form submission does not work, would you follow the get started button on [this link](https://formspree.io/html/) to learn how you should use formspree? , actually the `action` value should be similar to `https://formspree.io/f/{form_id}`

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @ManarOmari , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ManarOmari) [reply index.html]: Please , can you review my pull request ?

Comment (ManarOmari) [reply index.html]: i have created account on Formspree and update the action of my form

**Review by omar-labana — APPROVED** [review]
# **Approved ✔️**

Hi @ManarOmari

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**